### PR TITLE
Add mip dna skeleton

### PR DESCRIPTION
**Pull request by seallard**

### CODE REVIEW

File: cg/services/order_validation_service/workflows/mip_dna/constants.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+from cg.constants.constants import DataDelivery
+
+
+class TomteDeliveryType(Enum):
```
Comment (islean) [inline cg/services/order_validation_service/workflows/mip_dna/constants.py:6]: wrong workflow

File: cg/services/order_validation_service/workflows/mip_dna/constants.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    ANALYSIS_FILES = DataDelivery.ANALYSIS_FILES
+    FASTQ = DataDelivery.FASTQ
+    FASTQ_ANALYSIS = DataDelivery.FASTQ_ANALYSIS
+    NO_DELIVERY = DataDelivery.NO_DELIVERY
```
Comment (islean) [inline cg/services/order_validation_service/workflows/mip_dna/constants.py:10]: Do we want to fix these in this PR?

File: cg/services/order_validation_service/workflows/mip_dna/validation/field/model_validator.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+from pydantic_core import ValidationError
+
+from cg.services.order_validation_service.errors.validation_errors import ValidationErrors
+from cg.services.order_validation_service.validators.field.utils import convert_errors
+from cg.services.order_validation_service.workflows.mip_dna.models.order import MipDnaOrder
+
+
+class TomteModelValidator:
```
Comment (islean) [inline cg/services/order_validation_service/workflows/mip_dna/validation/field/model_validator.py:8]: Wrong workflow

File: cg/services/order_validation_service/workflows/mip_dna/validation_service.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+    def __init__(self, store: Store):
+        self.store = store
+
+    def validate(self, raw_order: dict) -> dict:
+        errors: ValidationErrors = self._get_errors(raw_order)
+        return create_order_validation_response(raw_order=raw_order, errors=errors)
+
+    def _get_errors(self, raw_order: dict) -> ValidationErrors:
+        order, field_errors = TomteModelValidator.validate(raw_order)
```
Comment (islean) [inline cg/services/order_validation_service/workflows/mip_dna/validation_service.py:35]: Wrong workflow

**Review by islean — APPROVED** [review]
Would we not want to change the BaseModel structure here though and introduce the OrderWithCases class?

File: cg/services/order_validation_service/workflows/mip_dna/constants.py (hunk 2 above)
Comment (seallard) [reply cg/services/order_validation_service/workflows/mip_dna/constants.py]: No!

**Pull request merged**